pkg/plugins/manager/store/store_test.go

```diff
@@ -42,7 +42,8 @@ func TestStore_ProvideService(t *testing.T) {
 			}
 		}}
 
-		_, err := ProvideService(fakes.NewFakePluginRegistry(), srcs, l)
+		s, err := ProvideService(fakes.NewFakePluginRegistry(), srcs, l)
+		s.starting(context.Background())
```

Comment:
Should we be calling StartAsync and deferring StopAsync here?

Reply:
updated, thanks!